symplyphysics/laws/electricity/charge_is_quantized.py

```diff
@@ -0,0 +1,35 @@
+from sympy import Eq, solve, Derivative
+from symplyphysics import (
+    Symbol,
+    units,
+    dimensionless,
+    print_expression,
+    Quantity,
+    validate_input,
+    validate_output,
+)
+
+# Description
+## Electric charge is quantized, i.e. restricted to certain values.
+
+# Law: q = n*e
+## q - electric charge
+## n - an integer factor
+## e - elementary charge
+
+charge = Symbol("charge", units.charge)
+integer_factor = Symbol("factor", dimensionless)
+
+law = Eq(charge, integer_factor * units.elementary_charge)
```

Comment:
It assumes that any charge can only be negative (or positive, depending on elementary charge definition). Probably we should tell about charge magnitude and use absolute values.

Reply:
As far as I know, most of the time in the English terminology the word `integer` refers to both positive and negative numbers. It would probably suffice to add a comment about that.

Reply:
I do not think that proton has negative amount of charges.

Reply:
I think it should be signed. Can you give me a link to statement that elementary charge is always positive?

Reply:
In SI units it is defined as such: https://www.bipm.org/utils/common/pdf/si-brochure/SI-Brochure-9.pdf

![image](https://github.com/blackyblack/symplyphysics/assets/115647936/497ff994-6310-4f1f-87aa-ee7cb22622c7)

Translating from French: ...It is defined taking the fixed numeric value of the elementary charge e equal to 1,602... ... This definition implies the exact relation e = 1,602...

Reply:
Thanks.